dashboard/test/controllers/lti_v1_controller_test.rb

```diff
@@ -0,0 +1,59 @@
+require "test_helper"
+
+class LtiV1ControllerTest < ActionDispatch::IntegrationTest
+  setup_all do
+    @integration = create :lti_integration
+  end
+
+  test 'given no params, return unauthorized' do
+    get '/lti/v1/login', params: {}
+    assert_response :unauthorized
+  end
+
+  test 'given a client_id that doesn not exist, return unauthorized' do
+    get '/lti/v1/login', params: {client_id: 'nope', iss: @integration.issuer}
+    assert_response :unauthorized
+  end
+
+  test 'given a platform_id that does not exist, return unauthorized' do
+    get '/lti/v1/login/wrong-id', params: {}
+    assert_response :unauthorized
+  end
+
+  test 'given a valid client_id, return redirect' do
```

Comment:
I would update the test string to include 'via GET' since the next test has the 'via POST'